node/standalone/src/chain_spec.rs

```diff
@@ -134,6 +135,7 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 				get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
 			],
 			true,
+			1282, // ChainId
```

Comment:
Doesn't seem to match the chainId from the readme

Reply:
The readme says
```
Local standalone testnet 1281
Reserved for other testnets 1282 - 1283
```

The standalone testnet is the "development" spec where I coded 1281. This is a multi-node local testnet which I categorized as "other testnets"/

Reply:
Ok I thought it was the same as the standalone one (with multi-node like the standalone can do)

Reply:
Oh great point. I was thinking of standalone in this context as meaning single-node. But we totally use it elsewhere to mean not-connected-to-relay-chain.

Reply:
Yeah, I actually agree with you. Let's change it.
```suggestion
			1281, // ChainId
```